Sources/JobsPostgres/Migrations/CreateJobDelay.swift

```diff
@@ -0,0 +1,55 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Hummingbird server framework project
+//
+// Copyright (c) 2024 the Hummingbird authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See hummingbird/CONTRIBUTORS.txt for the list of Hummingbird authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import HummingbirdPostgres
+import Logging
+import PostgresNIO
+
+struct CreateJobDelay: PostgresMigration {
+    func apply(connection: PostgresConnection, logger: Logger) async throws {
+        try await connection.query(
+            """
+            ALTER TABLE _hb_pg_job_queue ADD COLUMN IF NOT EXISTS delayed_until TIMESTAMP WITH TIME ZONE
+            """,
+            logger: logger
+        )
+        try await connection.query(
+            """
+            ALTER TABLE _hb_pg_jobs ADD COLUMN IF NOT EXISTS delayed_until TIMESTAMP WITH TIME ZONE
+            """,
+            logger: logger
+        )
```

Comment:
See my comment in PostgresJobsQueue.swift about whether we need a delayed_until column in _hb_pg_jobs.

Reply:
I will drop the delay until from the jobs table.